src/dotnet/install.sh

```diff
@@ -193,6 +193,14 @@ install_using_apt() {
         # Import key safely and import Microsoft apt repo
         curl -sSL ${MICROSOFT_GPG_KEYS_URI} | gpg --dearmor > /usr/share/keyrings/microsoft-archive-keyring.gpg
         echo "deb [arch=${architecture} signed-by=/usr/share/keyrings/microsoft-archive-keyring.gpg] https://packages.microsoft.com/repos/microsoft-${ID}-${VERSION_CODENAME}-prod ${VERSION_CODENAME} main" > /etc/apt/sources.list.d/microsoft.list
+
+        # See https://github.com/dotnet/core/issues/7699
```

Comment:
Do you think we should remove this after the install completes? I'm not sure if this will cause issues in subsequent `apt` operations.

Reply:
Yep, we can clean this! Updated